README.md

```diff
@@ -31,12 +33,27 @@ jobs:
 [...]
 ```
 
+#### Setting up Develocity for npm
+
+Inset the `Setup npm` step once in each job having steps invoking npm. Make sure to put the step before the npm invocation.
+
+```yaml
+name: PR Build
+jobs:
+  build:
+      - name: Setup npm
+        uses: gradle/develocity-actions/setup-npm@v1.5
```

Comment:
The version looks interesting :) Is it an arbitrary number?

Reply:
The current version is 1.4, so I assumed that the next version will be 1.5, and will be the first to include npm support.